packages/ember-glimmer/lib/component.js

```diff
@@ -109,13 +108,489 @@ export const BOUNDS = symbol('BOUNDS');
+    attributeBindings: ['disabled'],
+    disabled: Ember.computed(function() {
+      if (someLogic) {
+        return true;
+      } else {
+        return false;
+      }
+    })
+  });
+  ```
+  To prevent setting an attribute altogether, use `null` or `undefined` as the
+  return value of the `attributeBindings` monitored property:
+
+  ```app/components/my-text-input.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    tagName: 'form',
+    attributeBindings: ['novalidate'],
+    novalidate: null
+  });
+  ```
+  Updates to the property of an attribute binding will result in automatic
+  update of the  HTML attribute in the component's rendered HTML representation.
+  `attributeBindings` is a concatenated property. See [Ember.Object](/api/classes/Ember.Object.html)
+  documentation for more information about concatenated properties.
+
+
+  ## Layouts
+
+  Components can have a secondary template that wraps their main template. Like
+  primary templates, layouts can be any function that  accepts an optional
+  context parameter and returns a string of HTML that will be inserted inside
+  component's tag. Components whose HTML element is self closing (e.g. `<input />`)
+  cannot have a layout and this property will be ignored.
+  Most typically in Ember a layout will be a compiled template.
+  A components's layout can be set directly with the `layout` property or reference
+  an existing template by name with the `layoutName` property.
+  A template used as a layout must contain a single use of the
+  `{{yield}}` helper. The HTML contents of a components's rendered `template` will be
+  inserted at this location:
+
+  ```app/components/my-component-with-layout.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    layout: Ember.HTMLBars.compile("<div class='my-decorative-class'>{{yield}}</div>"),
+    template: Ember.HTMLBars.compile("I got wrapped")
+  });
+  ```
+  Will result in view instances with an HTML representation of:
+
+  ```html
+  <div id="ember1" class="ember-view">
+    <div class="my-decorative-class">
+      I got wrapped
+    </div>
+  </div>
+  ```
+  See [Ember.Templates.helpers.yield](/api/classes/Ember.Templates.helpers.html#method_yield)
+  for more information.
+
+
+  ## Responding to Browser Events
+
+  Components can respond to user-initiated events in one of three ways: method
+  implementation, through an event manager, and through `{{action}}` helper use
+  in their template or layout.
+  ### Method Implementation
+  Components can respond to user-initiated events by implementing a method that
+  matches the event name. A `jQuery.Event` object will be passed as the
+  argument to this method.
+
+  ```app/components/my-widget.js
+  import Ember from 'ember';
+
+  export default Ember.Component.extend({
+    click: function(event) {
+      // will be called when an instance's
+      // rendered element is clicked
+    }
+  });
+  ```
+
+  ### Event Managers
```

Comment:
Can you remove this section, but keep it handy somehow? We'll address it in a follow-up PR. The sets will be something like adding an `eventManager` property definition in `Ember.Component` and moving this documentation there.